source/Camera/Basic/CaptureHDRCompleteSettings/CaptureHDRCompleteSettings.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+This example shows how to acquire an HDR image from the Zivid camera with fully
+configured settings for each frame. In general, taking an HDR image is a lot
+simpler than this as the default settings work for most scenes. The purpose of
+this example is to demonstrate how to configure all the settings.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Connecting to the camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        std::cout << "Configuring settings same for all HDR frames" << std::endl;
+        auto settingsDefault = Zivid::Settings();
+        settingsDefault.set(Zivid::Settings::Brightness{ 1 })
+            .set(Zivid::Settings::Gain{ 1 })
+            .set(Zivid::Settings::Bidirectional{ false })
+            .set(Zivid::Settings::Filters::Contrast::Enabled::yes)
+            .set(Zivid::Settings::Filters::Contrast::Threshold{ 5 })
+            .set(Zivid::Settings::Filters::Gaussian::Enabled::yes)
+            .set(Zivid::Settings::Filters::Gaussian::Sigma{ 1.5 })
+            .set(Zivid::Settings::Filters::Outlier::Enabled::yes)
+            .set(Zivid::Settings::Filters::Outlier::Threshold{ 5 })
+            .set(Zivid::Settings::Filters::Reflection::Enabled::yes)
+            .set(Zivid::Settings::Filters::Saturated::Enabled::yes)
+            .set(Zivid::Settings::BlueBalance{ 1.081 })
+            .set(Zivid::Settings::RedBalance{ 1.709 });
+
+        std::cout << "Configuring settings different for all HDR frames" << std::endl;
+        size_t iris[3] = { 10U, 20U, 30U };
```

Comment:
Implement one of these here: https://zividlabs.atlassian.net/browse/CS-266

This means, change values for iris and exposureTime arrays, and add a gain array as well.